Minesweeper.py

```diff
@@ -43,68 +28,86 @@
                 print(board[y][x], end=" ")
         print()
 
-    # Print the game stats
+def print_stats():
     print("Moves:", moves)
     print("Flags:", flags)
     print("Mines found:", mines_found)
 
-    # Get the player's input
-    action = input("Enter your action (move, flag, or unflag): ")
-    direction = input("Enter the direction (up, down, left, or right): ")
-
-    # Update the player's position
+def update_player_pos(direction):
+    x, y = player_pos
     if direction == "up":
-        player_y -= 1
+        y -= 1
     elif direction == "down":
-        player_y += 1
+        y += 1
     elif direction == "left":
-        player_x -= 1
+        x -= 1
```

Comment:
The absence of a newline at the end of the file could be an issue if future modifications rely on it. Please consider adding a newline here. ```diff -    \ No newline at end of file + ``` ---